compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -105,13 +105,26 @@ namespace TR { class RegisterDependencyConditions; }
 // Hack markers
 #define CANT_REMATERIALIZE_ADDRESSES(cg) (cg->comp()->target().is64Bit()) // AMD64 produces a memref with an unassigned addressRegister
 
+void TR_X86ProcessorInfo::reset()
+   {
+   _vendorFlags = 0;
+   _featureFlags = 0;
+   _featureFlags2 = 0;
+   _featureFlags8 = 0;
+   _processorDescription = 0;
+   }
 
-TR_X86ProcessorInfo OMR::X86::CodeGenerator::_targetProcessorInfo;
-
-void TR_X86ProcessorInfo::initialize()
+void TR_X86ProcessorInfo::initialize(bool force)
    {
-   if (_featureFlags.testAny(TR_X86ProcessorInfoInitialized))
+   if (force)
+      {
+      reset();
```

Comment:
have you tested with behaviour with CRIU?

Reply:
Yeah; in fact this change is needed to fix https://github.com/eclipse-openj9/openj9/issues/15814